### DISCUSSION TITLE
代码块点复制按钮没反应

### DISCUSSION THREAD

**Discussion by scdsskj** [hugo-fixit/FixIt#529] [Q&A] [answered]
已合并
[markup]
  _merge = "shallow"
[outputs]
  _merge = "shallow"
[taxonomies]
  _merge = "shallow"
已打开
[params.page.code]
      enable = true
      copy = true
      edit = true
      maxShownLines = 3
可是还是不行，其他需要配置哪里？

**Comment by Lruihao** [comment, marked as answer]
网站是http的协议吗？目前遗留的一个兼容性问题，复制功能只支持https协议或者本地调试预览。

**Reply by scdsskj** [reply]
我加上证书用https试一下 谢谢🙏

**Reply by Lruihao** [reply]
预计 v0.3.14 版本解决复制功能在 http 下兼容性问题。
